app/components/provider-links/template.hbs

```diff
@@ -0,0 +1,39 @@
+{{#if fetchData.isRunning}}
+    {{#content-placeholders as |placeholder|}}
+        <span>
+            <i class="fa-li icon--skeleton">{{placeholder.icon}}</i>
+            {{placeholder.heading subtitle=false}}
```

Comment:
It looks like it's still adding a subtitle, and it looks weird:
<img width="432" alt="screen shot 2017-11-14 at 3 11 54 pm" src="https://user-images.githubusercontent.com/6776190/32802913-d08219dc-c94f-11e7-931e-5d7c6122b29a.png">

Reply:
Did you run `yarn install`? I made a change to `ember-content-placeholders`.

Reply:
Oh whoops, I did not. All good 👍